### ISSUE TITLE
Matcha init error

### ISSUE THREAD

**Issue by Sebastianbrg** [gomatcha/matcha#31]
Getting this error when trying to run "matcha init"
![image](https://user-images.githubusercontent.com/4181050/31318248-6aa0e0ac-ac4f-11e7-959e-ed26d877e533.png)
xcrun --show-sdk-path: exit status 1

**Comment by heheue** [comment]
Just install XCode and open it. In case you have done it and still you can see the error, type `sudo xcode-select -r`. Glad I could help!

**Comment by overcyn** [comment]
Yeah what heheue said. If that doesn't work what does `xcrun --show-sdk-path` return?

**Comment by heheue** [comment]
`xcrun --show-sdk-path` should return path pointing to installed OSX's SDK, otherwise it will return null or some error code
